### PR TITLE
SDK 3.44.0-preview.1: Adds Version bump and updates changelog

### PR DESCRIPTION
**Pull request by NaluTripician**
# Pull Request Template
This bumps the SDK version 3.44.0-preview.1. There was no changes to the GA public contract.

Preview contract changes:

image

### CODE REVIEW

File: Microsoft.Azure.Cosmos/contracts/API_3.44.0-preview.1.txt [hunk 1]
```diff
@@ -0,0 +1,1583 @@
...
+    {
+        protected ChangeFeedEstimator();
+        public abstract FeedIterator<ChangeFeedProcessorState> GetCurrentStateIterator(ChangeFeedEstimatorRequestOptions changeFeedEstimatorRequestOptions=null);
+    }
+    public sealed class ChangeFeedEstimatorRequestOptions
+    {
+        public ChangeFeedEstimatorRequestOptions();
+        public Nullable<int> MaxItemCount { get; set; }
+    }
+    public abstract class ChangeFeedMode
```
Comment (philipthomas-MSFT) [inline Microsoft.Azure.Cosmos/contracts/API_3.44.0-preview.1.txt:42]: Why are we removing AllVersionsAndDeletes?

Comment (NaluTripician) [reply Microsoft.Azure.Cosmos/contracts/API_3.44.0-preview.1.txt:42]: I am not sure why the generateAPI script said that was removed. I will re-run it to fix this. Good catch thanks.

Comment (philipthomas-MSFT) [reply Microsoft.Azure.Cosmos/contracts/API_3.44.0-preview.1.txt:42]: ChangeFeedItem<T>, ChangeFeedMeta and ChangeFeedOperationType also missing?

Comment (philipthomas-MSFT) [reply Microsoft.Azure.Cosmos/contracts/API_3.44.0-preview.1.txt:42]: The screenshot in the description and the preview txt is showing that this is going away.

**Pull request closed without merging**